client.go

```diff
@@ -16,7 +16,13 @@ type Client interface {
 	Push() (string, error)
 	// Run runs the specified deploy, returning the port it is listening on.
 	Run(deployId string) (int, error)
+
+	// Sets active deploy by port
```

Comment:
Good pick, will fixup in a follow up commit (This change will touch basically every file)